xcode/Subconscious/Shared/Library/SQLite3Database.swift

```diff
@@ -209,10 +226,24 @@ final class SQLite3Database {
         }
     }
 
-    public enum SQLite3DatabaseError: Error {
+    public enum SQLite3DatabaseError: Error, LocalizedError {
```

Comment:
Conform to LocalizedError. This gives us more insight into what goes wrong when there is an error on the SQLite side.